notifications/notifications.go

```diff
@@ -20,8 +21,15 @@ func Send(client Client, endpoint string, msg string, headers map[string]string)
 func ReportSchedule(client Client, schedule *schedule.Schedule) bool {
 	success := true
 	receiver := config.NotificationsAttacks()
-	msg := fmt.Sprintf("{\"text\": \"schedule:\n%s\n\"}", schedule)
+	sched := "schedule"
 
+	kubeMonkeyID := os.Getenv("KUBE_MONKEY_ID")
+
+	if kubeMonkeyID != "" {
+		sched = fmt.Sprintf("schedule(id: %s)", kubeMonkeyID)
+	}
+
+	msg := fmt.Sprintf("{\"text\": \"%s:\n%s\n\"}", sched, schedule)
```

Comment:
I think there's a `}` missing at the end?

Reply:
@worldtiki hmm I can't see where `}` is missing in the end :/